bpf/aya-bpf/src/args.rs

```diff
@@ -56,6 +56,30 @@ unsafe_impl_from_btf_argument!(i64);
 unsafe_impl_from_btf_argument!(usize);
 unsafe_impl_from_btf_argument!(isize);
 
+pub struct PtRegs {
+    regs: *mut pt_regs,
```

Comment:
Users can’t take the pointer out of the struct in the current state. Need something like as_ptr.